contents/tutorials/bubble-surveys.md

```diff
@@ -0,0 +1,53 @@
+    posthog.init('<ph_api_key>',{api_host:'<ph_instance_address>'})
+</script>
+```
+
+With the snippet copied, go to your Bubble site settings by clicking on the icon in the left-hand menu. If you haven’t already, sign up for at least the "Starter" site plan. This enables you to add custom code.
+
+Go to the **SEO / metatags** tab in site settings. Paste your PostHog snippet in the **Script/meta tags in header** section. Then, deploy your site to live.
+
+![How to add PostHog to Bubble](../images/tutorials/bubble-surveys/adding-posthog.mp4)
+
+## 2. Create and launch your survey
+
+Create a survey in PostHog by going to the [surveys tab](https://app.posthog.com/surveys) and clicking "New survey." There are a variety of [survey types](/docs/surveys/creating-surveys#question-type) to choose from, or you can create your own by clicking "Create blank survey".
+
+![PostHog survey templates](../images/tutorials/bubble-surveys/survey-templates.png)
+
+Next, customize your survey as needed. You can customize the questions, branding, and targeting. See our [survey docs](/docs/surveys/creating-surveys) for more details on how to do so.
+
+Then, click "Save as draft" and then "Launch". Your survey is now live and you should see it on your website! 
+
+## 3. View results
+
+After interacting with your survey, you can view results by selecting the survey from the [surveys tab](https://app.posthog.com/surveys). You'll see data on:
+
+- How many users have seen the survey.
+- How many users have dismissed the survey.
+- Responses.
+
+You can also filter these results based on [user properties](/docs/product-analytics/user-properties), [cohorts](/docs/data/cohorts), [feature flags](/docs/feature-flags/creating-feature-flags) and more.
+
+![Survey results](../images/tutorials/bubble-surveys/survey-results.png)
+
+## Further reading
+
+- [How to analyze surveys with ChatGPT](/tutorials/analyze-surveys-with-chatgpt)
```

Comment:
Later this week Im adding 2 other tutorials for bubble: Analytics and A/B tests.

I'll add the links here when they ready (and add links to the tutorial on those tutorials)